models/favorite.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
As far as I know, it's not necessary to add this model, sequelize does it automatically. If you wanted to add extra attributes to the Favorite model, then we would usually create the model and have it here. 

Regardless, it's probably still good to have it to make the associations more explicit/ for readability. 

Just a fun fact

Reply:
How interesting. Cool to know. Thank you!